### PR TITLE
Social packages test

### PR DESCRIPTION
**Pull request by rwhite27**
New features:

- Create and Delete Messages
- Create, Edit and Delete comments from messages

### CODE REVIEW

**Comment by FredPeal** [comment]
@rwhite27 the test is not success

File: api/controllers/FeedsController.php [hunk 1]
```diff
@@ -39,29 +48,116 @@ public function create() : Response
...
+        $validation->add('leads_uuid', new PresenceOf(['message' => _('leads_uuid is required.')]));
+        $validation->add('message', new PresenceOf(['message' => _('message is required.')]));
+        $validation->add('verb', new PresenceOf(['message' => _('verb is required.')]));
+        $validation->validate($request);
+
+        // Poner en creacion de lead que se cree tambien el channell
         $lead = LeadsModel::findFirstOrFail([
-            'conditions' => 'uuid = ?0',
-            'bind' => [$id]
+            'conditions' => 'uuid = :uuid:',
```
Comment (kaioken) [inline api/controllers/FeedsController.php:129]: @rwhite27 look it up as a relationship could also be a solution , but do change the comments to english

**Review by kaioken — CHANGES_REQUESTED** [review]

File: api/controllers/FeedsController.php (hunk 1 above)
Comment (rwhite27) [reply api/controllers/FeedsController.php:128]: done. Now lead's channel is fetched via the relationship between leads and channels.

File: phinx-social.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    'paths' => [
+        'migrations' => 'vendor/kanvas/packages/storage/db/Social/migrations',
+        'seeds' => 'vendor/kanvas/packages/storage/db/Social/seeds',
+    ],
+    'environments' => [
+        'default_migration_table' => 'ut_migrations',
+        'default_database' => 'development',
+        'production' => [
+            'adapter' => 'mysql',
+            'host' => getenv('DATA_API_SOCIAL_MYSQL_HOST'),
```
Comment (FredPeal) [inline phinx-social.php:16]: add this new variables to .env.example

**Pull request merged**